web-ng/src/app/services/project/project.service.ts

```diff
@@ -137,5 +137,25 @@ export class ProjectService {
       .toList()
       .sortBy(entry => entry.email)
       .toArray();
+   }
+  /** 
+   * Checks if a user can add points to a specific layer.
+   */
+  canUserAddPointsToLayer(layer: Layer): boolean {
```

Comment:
Can this method be moved into AuthService?

Reply:
This function needs `current project`, if we move this to AuthService, we can't use ProjectService in AuthService, because AuthService is already used in ProjectService. So we could move this to AuthService and change the signature to `canUserAddPointsToLayer(project: Project, layer: Layer): boolean`, Wdyt?

Reply:
updated.